content/docs/custom-plugins/authenticated-proxy/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Configuring Authenticated Proxy for Custom Plugins
+lastUpdated: '2023-02-03T10:27:00.0Z'
+description: How to configure a custom proxy to identify the Backstage user for you custom plugin.
+---
+
+## Introduction
+
+Roadie provides a custom proxy service that can be configured to proxy through an access token minted by our own authentication system that can be validated by your third party service.
+
+
+## Step 1. Create an authenticated proxy configuration in Roadie
+
+You can navigate to https://your-company.roadie.so/administration/settings/authenticatedProxy. Click add item to add a new proxy.
```

Comment:
It's best to not let these links become actual links. Otherwise they become clickable and it's annoying to click it and go to a 404.

```suggestion
You can navigate to /administration/settings/authenticatedProxy. Click add item to add a new proxy.
```

Reply:
Done.